### ISSUE TITLE
duplicate in Artsdata

### ISSUE THREAD

**Issue by troughc** [culturecreates/nebula#38]
https://kg.artsdata.ca/en/entity?uri=http%3A%2F%2Fkg.artsdata.ca%2Fresource%2FK5-993
is the same as
https://kg.artsdata.ca/en/entity?uri=http%3A%2F%2Fkg.artsdata.ca%2Fresource%2FK5-237

Unrelated question :)
I noticed that [one of my test places](https://kg.artsdata.ca/en/entity?uri=http%3A%2F%2Flod.footlight.io%2Fresource%2F66d8c6113451310064481d35) is in artsdata. It is NOT minted. I just deleted it from the CMS. My question is, will it be deleted from Artsdata?

**Comment by saumier** [comment]
> Unrelated question :)
> I noticed that [one of my test places](https://kg.artsdata.ca/en/entity?uri=http%3A%2F%2Flod.footlight.io%2Fresource%2F66d8c6113451310064481d35) is in artsdata. It is NOT minted. I just deleted it from the CMS. My question is, will it be deleted from Artsdata?

@troughc Your test place was uploaded in the data feed into an external graph of Artsdata. It will be gone the next time the data feed from CMS is uploaded. No problem. Only minted data with the authority domain artsdata.ca will remain in Artsdata and be sent out to people using Artsdata APIs to query or reconcile.  Since your entity http://lod.footlight.io/resource/66d8c6113451310064481d35 is not using the domain artsdata.ca it will not be sent out to people using Artsdata.

**Comment by troughc** [comment]
> @troughc Thanks for sharing this duplicate. I already have it on my list of all duplicates. There is a query available on the home page of Artsdata.ca: [Same name people-org-place](https://kg.artsdata.ca/query/show?sparql=https://raw.githubusercontent.com/artsdata-stewards/artsdata-actions/main/queries/same_name_people-org-place.sparql)
> 
> I am not fixing them yet because I want to add the button for Tammy so someone else can fix them.

Thank @saumier If I find other duplicates i will check your list. Is there somewhere more permanent that I would be able to find the link? If not I will look in this ticket. cc @troughc

**Comment by saumier** [comment]
@troughc Yes. As I mentioned, the list of common queries is on your home page of Artsdata.ca. You must be logged in.
